docker-compose.yml

```diff
@@ -31,7 +31,6 @@ services:
     frontend:
       volumes:
         - ./frontend/src:/usr/src/app/src
-        - .env:/usr/src/app/.env
```

Comment:
You would still need to load it no?

Reply:
So, not quite. This line doesn't do anything as far as I can tell - I think `volumes` only works for folders, not for files. Here's how it works now:

- For local development (`yarn watch`), we load the .env file from the root directory and read `MAPBOX_TOKEN` from there ([code](https://github.com/bayesimpact/tds/blob/143d534aaff3831db522ff0b67af36715bf1c042/frontend/webpack.config.js#L52))
- For production (`docker-compose build frontend`/`docker-compose up frontend`), we read the .env from the root directory and pass everything in it as env vars to the frontend Dockerfile ([code](https://github.com/bayesimpact/tds/blob/143d534aaff3831db522ff0b67af36715bf1c042/docker-compose.yml#L40)), read the `MAPBOX_TOKEN` env var in the frontend Dockerfile and pass it to the webpack config ([code](https://github.com/bayesimpact/tds/blob/143d534aaff3831db522ff0b67af36715bf1c042/frontend/Dockerfile#L3)), and finally pass it from the webpack config to our code ([code](https://github.com/bayesimpact/tds/blob/143d534aaff3831db522ff0b67af36715bf1c042/frontend/webpack.config.prod.js#L70)). All of this is to get around the restriction that docker's `env_file` directive can't load .env files from parent directories for security reasons, and to avoid having to maintain 2 separate .env files (one in the root dir, and one in the frontend dir).

Reply:
sounds good